src/sagemaker/local/image.py

```diff
@@ -720,6 +909,16 @@ def _check_output(cmd, *popenargs, **kwargs):
     return output
 
 
+def _create_processing_config_file_directories(root, host):
+    """
+    Args:
+        root:
+        host:
+    """
```

Comment:
either remove or complete the docstring for private methods (i tend to favor completion)

Reply:
Added docstrings.